src/saveload/afterload.cpp

```diff
@@ -1428,12 +1434,12 @@ bool AfterLoadGame()
 		for (Station *st : Station::Iterate())   st->build_date      += DAYS_TILL_ORIGINAL_BASE_YEAR;
 		for (Waypoint *wp : Waypoint::Iterate()) wp->build_date      += DAYS_TILL_ORIGINAL_BASE_YEAR;
 		for (Engine *e : Engine::Iterate())      e->intro_date       += DAYS_TILL_ORIGINAL_BASE_YEAR;
-		for (Company *c : Company::Iterate()) c->inaugurated_year += ORIGINAL_BASE_YEAR;
-		for (Industry *i : Industry::Iterate())  i->last_prod_year   += ORIGINAL_BASE_YEAR;
+		for (Company *c : Company::Iterate())    c->inaugurated_year += static_cast<TimerGameEconomy::Year>(ORIGINAL_BASE_YEAR);
+		for (Industry *i : Industry::Iterate())  i->last_prod_year   += static_cast<TimerGameEconomy::Year>(ORIGINAL_BASE_YEAR);
 
 		for (Vehicle *v : Vehicle::Iterate()) {
-			v->date_of_last_service += DAYS_TILL_ORIGINAL_BASE_YEAR;
-			v->build_year += ORIGINAL_BASE_YEAR;
+			v->date_of_last_service += static_cast<TimerGameEconomy::Year>(DAYS_TILL_ORIGINAL_BASE_YEAR);
+			v->build_year += static_cast<TimerGameEconomy::Year>(ORIGINAL_BASE_YEAR);
```

Comment:
These kind of things shouldn't exist :D I like StrongTypeDef .. shows when you are cheating :D